app/src/main/res/layout/activity_gather.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".GatherActivity">
+
+    <ImageView
+        android:id="@+id/gatherAnimation"
+        android:layout_width="122dp"
+        android:layout_height="200dp"
+        android:visibility="visible"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.498"
+        app:srcCompat="@android:drawable/btn_star_big_on" />
+</android.support.constraint.ConstraintLayout>
```

Comment:
No biggie, but do we want new line at end of file?

Reply:
Not quite sure what you mean here, do you want me to end the section at the previous line?
There is not an empty line at the bottom of the file in my editor at least

Reply:
The question is if we should have a new line at the end of the file... it's convention in a lot of places.
Google gave me this: https://stackoverflow.com/a/729795/2148380